Adyen.EcommLibrary/Service/ServiceResource.cs

```diff
@@ -20,17 +20,15 @@ public string Request(string json)
         {
             var clientInterface = _abstractService.Client.HttpClient;
             var config = _abstractService.Client.Config;
-            var result = string.Empty;
-
+           
             try
             {
-                result = clientInterface.Request(Endpoint, json, config);
+                return clientInterface.Request(Endpoint, json, config);
             }
-            catch (HttpClientException httpClientException)
+            catch (Exception httpClientException)
             {
                 throw httpClientException;

```

Comment:
should we not split it up to exception and HttpClientException ?

Reply:
I do not understand the question. Would you like to explain me more? The exception at that point maybe is more generic so it can be like 
 ```   
catch (Exception exception)
            {
                throw exception;
            }
```